### ISSUE TITLE
PHP8 error with content-type reading

### ISSUE THREAD

**Issue by liayn** [helhum/typoscript_rendering#86]
```
[WARNING] request="b2b9....." component="TYPO3.CMS.Core.Error.ErrorHandler": Core: Error handler (FE): PHP Warning: Undefined array key 0 in .../vendor/helhum/typoscript-rendering/Classes/Middleware/TypoScriptRenderingMiddleware.php line 76
```

This is here: https://github.com/helhum/typoscript_rendering/blob/main/Classes/Middleware/TypoScriptRenderingMiddleware.php#L76

I'm not yet sure, how this happens, but maybe a check there would make sense anyways?

(The access log tells me the causing request was done via IPv6, so maybe on our way through our extension this somehow matters.)

**Comment by liayn** [comment]
I can now reproduce the problem. This happens, if an action does
`throw new PropagateResponseException(new RedirectResponse($redirect, 303), 16155538)`

So we get a response that does not have a content-type indeed.

**Comment by helhum** [comment]
Should be resolved with 2.4.1
